src/renderer/privacy.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import styles from './privacy.css';
+
+const PrivacyElement = () => (
+    <div className={styles.privacyBox}>
+        <h1>Privacy Policy</h1>
+        <p>
+            We understand how important privacy is to our community, especially children and their parents. We wrote
+            this privacy policy to explain what information we collect through the Scratch application (the
+            &ldquo;App&rdquo;), how we use it, and what we&apos;re doing to keep it safe. If you have any questions
+            regarding this privacy policy, you can <a
+                href="https://scratch.mit.edu/contact-us/"
+                target="_blank"
+                rel="noopener noreferrer"
+            >contact us</a>.
```

Comment:
I think it's OK for this to be a web site link since you won't be able to contact us unless you have Internet access anyway.